gulpfile.mjs

```diff
@@ -333,7 +335,28 @@ function createWebpackConfig(
     mode: "production",
     optimization: {
       mangleExports: false,
-      minimize: false,
+      minimize: isMinified,
+      minimizer: !isMinified
+        ? undefined
+        : [
+            new TerserPlugin({
+              extractComments: false,
+              parallel: false,
```

Comment:
Is there a reason why we can't set this to `true` instead to speed up minimization? Looking at https://webpack.js.org/plugins/terser-webpack-plugin/#parallel it also says setting this to `true` is the recommended settings.

Reply:
That page contains the following text, hence I'd rather not risk this potentially breaking in GitHub Actions (since that could break the next release):
> Warning
>    If you use Circle CI or any other environment that doesn't provide real available count of CPUs then you need to setup explicitly number of CPUs to avoid Error: Call retries were exceeded (see [#143](https://github.com/webpack-contrib/terser-webpack-plugin/issues/143), [#202](https://github.com/webpack-contrib/terser-webpack-plugin/issues/202)).

Reply:
Ah, yes, that seems like a good idea. Somehow I missed this warning text because I thought it exclusively applied to Circle CI, but it also mentions other virtualized environments.